src/Module/Photo.php

```diff
@@ -285,7 +285,7 @@ private static function getPhotoById(int $id, string $type, int $customsize)
 					return false;
 				}
 
-				if (Network::isLocalLink($url) && preg_match('|.*?/photo/(.*[a-fA-F0-9])\-(.*[0-9])\..*[\w]|', $url, $matches)) {
+				if (DI::baseUrl()->isLocalUrl($url) && preg_match('|.*?/photo/(.*[a-fA-F0-9])\-(.*[0-9])\..*[\w]|', $url, $matches)) {
```

Comment:
`BaseModule` has a `BaseUrl` dependency so you can avoid using `DI::baseUrl()` in the child classes of `BaseModule`.
```suggestion
				if ($this->baseUrl->isLocalUrl($url) && preg_match('|.*?/photo/(.*[a-fA-F0-9])\-(.*[0-9])\..*[\w]|', $url, $matches)) {
```